test/regression/pom.xml

```diff
@@ -777,5 +786,176 @@
                 </plugins>
             </build>
         </profile>
+		
+		<profile>
+			<id>new-jdk-regression-test</id>
+			<activation>
+				<jdk>[9.0,)</jdk>
```

Comment:
Given now some tests are scoped to JDK version I think the github runner needs to run twice (once with JDK8, once with e.g. JDK11).
Also as an aside I think the tmate lines in https://github.com/JacORB/JacORB/blob/master/.github/workflows/maven.yml#L41 should be commented out.

Reply:
For extending the CI I created https://github.com/JacORB/JacORB/issues/316, something we scheduled for after the merge of this PR

Reply:
I think we need a successful run from another PR to verify the amount of test differences.

Reply:
Yes, on my todo list, at the moment we created this PR using multiple java versions within one action was problematic

Reply:
Ok. It still runs the same amount of tests using JDK8 but I think there does need to be verification under a later JDK. I guess that could be pushed to a subsequent PR and there would also be the debate about what is a suitable JDK version to use? ( 11? 17?)

Reply:
With a matrix we could easily test multiple jdk versions